medmodels/data_synthesis/mtgan/model/loaders.pyi

```diff
@@ -0,0 +1,61 @@
+from typing import Dict, Optional
+
+import numpy as np
+import torch
+from numpy.typing import NDArray
+from torch.utils.data import DataLoader, Dataset
+
+from medmodels.data_synthesis.mtgan.modules.preprocessor import PreprocessingAttributes
+from medmodels.medrecord.medrecord import MedRecord
+from medmodels.medrecord.types import Group, MedRecordValue
+
+class MTGANDataset(Dataset[torch.Tensor]):
+    medrecord: MedRecord
+    device: torch.device
+
+    patients_group: Group
+    concepts_group: Group
+
+    preprocessing_attributes: PreprocessingAttributes
+
+    total_number_of_concepts: int
+    maximum_number_of_windows: int
+    concept_to_index_dict: Dict[MedRecordValue, int]
+
+    def __init__(
+        self,
+        medrecord: MedRecord,
+        patients_group: Group,
+        concepts_group: Group,
+        preprocessing_attributes: PreprocessingAttributes,
+        index_to_concept_dict: Dict[int, MedRecordValue],
```

Comment:
Are `concept_to_index_dict` and `index_to_concept_dict` the same or does this indicate we flip the key/values?

Reply:
We are flpping the key values inside the class, exactly.